watcher/watcher.go

```diff
@@ -424,13 +420,7 @@ func (watcher Watcher) burnToParams(txid pack.Bytes, amount pack.U256, toBytes [
 	var to multichain.Address
 	var toDecoded []byte
 	var err error
-	burnChain := watcher.selector.Source()
-	switch burnChain {
-	case multichain.Solana:
-		to, toDecoded, err = watcher.handleAssetAddrSolana(toBytes)
-	default:
-		to, toDecoded, err = watcher.handleAssetAddrEth(toBytes)
-	}
+	to, toDecoded, err = watcher.handleAssetAddrEth(toBytes)
```

Comment:
Should we rename this function if it's no longer just for Ethereum?

Reply:
Good point